jstz_node/src/services/accounts.rs

```diff
@@ -29,11 +32,67 @@ async fn nonce(
     Ok(HttpResponse::Ok().json(nonce))
 }
 
+#[get("/{address}/kv")]
+async fn kv(
+    rollup_client: Data<RollupClient>,
+    path: Path<String>,
+    query: web::Query<HashMap<String, String>>,
+) -> Result<impl Responder> {
+    let address = path.into_inner();
+    let key = query.get("key").cloned().unwrap_or_else(|| "".to_string());
+
+    let storage_key = if key == "" {
+        format!("/jstz_kv/{}", address)
+    } else {
+        format!("/jstz_kv/{}/{}", address, key)
+    };
```

Comment:
Nit: instead of using a sentinel value (`""`) to indicate empty, we could use the `Some/None` returned by `query.get`